react-refresher/my-app/src/Counter.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+
+export default function Counter({ initialNum }) {
+  const [count, setCount] = useState(Number(initialNum));
+  const handleCount = (event) => {
+    event.target.name === 'btn-down'
```

Comment:
This works, but it means that the function has to know the `name` prop of the button elements. It's best not to have that kind of knowledge spread around the code. If you instead passed the `handleCount` function the value it should increment by, you can make the code more reusable.

For example, `<button onClick={() => handleCount(-1)}>`
or even better, `<button onClick={() => setCount(count - 1)}>`.